pkg/skaffold/deploy/helm.go

```diff
@@ -214,8 +213,83 @@ func (h *HelmDeployer) Cleanup(ctx context.Context, out io.Writer) error {
 }
 
 // Render generates the Kubernetes manifests and writes them out
-func (h *HelmDeployer) Render(context.Context, io.Writer, []build.Artifact, []Labeller, string) error {
-	return errors.New("not yet implemented")
+func (h *HelmDeployer) Render(ctx context.Context, out io.Writer, builds []build.Artifact, labellers []Labeller, filepath string) error {
```

Comment:
I'm not well-versed in the renderer, but doesn't it seems very wrong for a function to accept both a `io.Writer` and a `filepath` that overrides the `io.Writer` argument.

 Is this just an API quirk you are trying to deal with?

Reply:
I am no versed enough in go to know if it is a good pattern or not.

All I can say is that the `filepath` parameter is used to support the `--output` option.
That decides whether you want your rendered manifest into a file or to the standard output.

You can see a similar usage as my PR [here](https://github.com/GoogleContainerTools/skaffold/blob/bb7807edd302bb6a354434037684a218b248f27e/pkg/skaffold/deploy/util.go#L86) and it usages in the `kubectl` renderer and [here](https://github.com/GoogleContainerTools/skaffold/blob/bb7807edd302bb6a354434037684a218b248f27e/pkg/skaffold/deploy/kustomize.go#L200) for the `kustomize` renderer.